about.html

```diff
@@ -7,67 +7,90 @@
-        
-      <footer>
+<main>
+    <header>
+        <img src = "img/lambda-black.png" id="lambdalogo">
         <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
         </nav>
-      </footer>
-    </div><!-- container -->        
-</body>
+      
+    </header>        
+    <img src="img/jumbo-about.png" id="jumbo-about"> 
+  <body>
+    <div class="container about-page">
+      <div class= "about-section">
+        <h1>About</h1>
+
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+        <br>
+      </div>
+      <div id="row1">
+          <div class="mid-tent" id="sec1"> 
+           <img src="img/about-plan.png" alt="strategy">
+  
+            <h2>Strategy</h2>
+            
+              <p class="midp">Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+              <button>Learn More</button>
+        </div>
+        <div class="mid-tent" id="sec2">    
+          <img src="img/about-working.png" alt="strategy">
+           
+            <h2>How We Work</h2>
+            
+              <p class= "midp">Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+              <button>Learn More</button>
+        </div>
+      </div>
+      <div id="row2">
+        <div class="mid-tent" id= "sec3">  
+          <img src="img/about-office.png" alt="strategy">
+          
+            <h2>Places We Work</h2>
+
+              <p class="midp">Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+              <button>Learn More</button>
+          </div>
+          <div class="mid-tent" id="sec4">  
+            <img src="img/about-meeting.png" alt="strategy">
+          
+            <h2>Collaboration</h2>
+            
+              <p class="midp">Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+              <button>Learn More</button>
+          </div>    
+      </div>
```

Comment:
might be a good idea to use more semantic HTML (like sections) to better match index.html